e2e/clientCanSignUp.e2e.js

```diff
@@ -0,0 +1,45 @@
+const { reloadApp } = require("detox-expo-helpers");
+const randomize = Math.floor(Math.random() * 10000);
+describe("Client can sign up", () => {
+  beforeEach(async () => {
+    await reloadApp();
+  });
+
+  it("successfully", async () => {
+    await expect(element(by.id("wantToPublishButton"))).toBeVisible();
+    await element(by.id("wantToPublishButton")).tap();
+    await element(by.id("emailInput")).tap();
+    await element(by.id("emailField")).typeText(`user${randomize}@gmail.com`);
+    await element(by.id("passwordLabel")).tap();
+    await element(by.id("passwordInput")).typeText("password");
+    await element(by.id("passwordConfirmationLabel")).tap();
+    await element(by.id("passwordConfirmationInput")).typeText("password");
+    await element(by.id("companyNameLabel")).tap();
+    await element(by.id("companyNameInput")).typeText("develup");
+    await element(by.id("companyUrlLabel")).tap();
+    await element(by.id("companyUrlInput")).typeText("develup.com");
+    await element(by.id("mainContainer")).tap();
+    await expect(element(by.id("submitButton"))).toBeVisible();
+    await element(by.id("submitButton")).tap();
+    await expect(element(by.id("welcomeMessage"))).toBeVisible();
+    await expect(element(by.id("createAssignmentButton"))).toBeVisible();
+  });
+  it("unsuccessfully", async () => {
+    await expect(element(by.id("wantToPublishButton"))).toBeVisible();
+    await element(by.id("wantToPublishButton")).tap();
+    await element(by.id("emailInput")).tap();
```

Comment:
same here, unsuccessfully with non matching password
and then change the password conf input